.github/workflows/docker.yml

```diff
@@ -29,3 +29,10 @@ jobs:
           v=${GITHUB_REF:10}
           docker tag brew "docker.pkg.github.com/homebrew/brew/ubuntu${{matrix.version}}:$v"
           docker push "docker.pkg.github.com/homebrew/brew/ubuntu${{matrix.version}}:$v"
+      - name: Deploy the tagged Docker image to Docker Hub
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          docker login docker.com -u brewtestbot -p ${{secrets.DOCKER_TOKEN}}
+          v=${GITHUB_REF:10}
```

Comment:
Might be nice to use https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable so this can be reused between steps

Reply:
Done.